Test/Unit/PaymentorderTest.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+
+namespace SwedbankPay\Checkout\Test\Unit;
+
+use Magento\Checkout\Model\Session as CheckoutSession;
+use Magento\Cms\Helper\Page as PageHelper;
+use Magento\Directory\Model\Currency;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Locale\Resolver;
+use Magento\Framework\UrlInterface;
+use Magento\Quote\Model\Quote;
+use Magento\Quote\Model\Quote\Address;
+use Magento\Store\Model\Store;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Theme\Block\Html\Header\Logo as HeaderLogo;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use SwedbankPay\Api\Client\Client as ApiClient;
+use SwedbankPay\Api\Service\Paymentorder\Resource\Collection\ItemsCollection;
+use SwedbankPay\Api\Service\Paymentorder\Resource\Collection\PaymentorderItemsCollection;
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderCampaignInvoice;
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderInvoice;
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderObject;
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderPayeeInfo;
+use SwedbankPay\Api\Service\Paymentorder\Resource\PaymentorderSwish;
+use SwedbankPay\Checkout\Helper\Config as PaymentMenuConfig;
+use SwedbankPay\Checkout\Helper\Paymentorder;
+use SwedbankPay\Checkout\Model\QuoteFactory;
+use SwedbankPay\Checkout\Model\ResourceModel\QuoteRepository;
+use SwedbankPay\Core\Helper\Config as ClientConfig;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+class PaymentorderTest extends TestCase
```

Comment:
Codacy found an issue: [The class PaymentorderTest has a coupling between objects value of 25. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-checkout/pullRequest?prid=5821673)

Reply:
Perhaps add a comment to ignore whatever rule Codacy thinks is broken here?